File: src/main/java/org/taktik/icure/services/external/rest/v1/dto/embed/AgreementAppendixDto.java
package org.taktik.icure.services.external.rest.v1.dto.embed;

public class AgreementAppendixDto {
	private Integer docSeq;
	private Integer verseSeq;
	private String documentId;
	private String path;

	public Integer getDocSeq() {
		return docSeq;
	}

	public void setDocSeq(Integer docSeq) {
		this.docSeq = docSeq;
	}

	public Integer getVerseSeq() {
		return verseSeq;
	}

	public void setVerseSeq(Integer verseSeq) {
		this.verseSeq = verseSeq;
	}

	public String getDocumentId() {
		return documentId;
	}

	public void setDocumentId(String documentId) {
		this.documentId = documentId;
	}

	public String getPath() {
		return path;
	}

	public void setPath(String path) {
		this.path = path;
	}
}

File: src/main/kotlin/org/taktik/icure/services/external/rest/v1/dto/embed/AgreementAppendixDto.kt
package org.taktik.icure.services.external.rest.v1.dto.embed

import java.io.Serializable
import com.github.pozo.KotlinBuilder
import com.github.pozo.KotlinBuilder
@KotlinBuilder
data class AgreementAppendixDto(
        val docSeq: Int? = null,
        val verseSeq: Int? = null,
        val documentId: String? = null,
        val path: String? = null
) : Serializable
